launchers/ids-connector/gradle.properties

```diff
@@ -0,0 +1,30 @@
+javaVersion=11
```

Comment:
Is this file needed?

Reply:
Hi, it was the only way I got the Dockerfile working. Even if I called the Docker build command from the project root, the build still failed until the files were moved to the launcher directory. Maybe I also missed something there since I'm not familiar with Gradle and Docker. If we can get it working another way, I would remove the files again.

Reply:
That makes sense. I'll try it out and then update the README and remove the files.

Reply:
though this might be different when calling Docker run form the project root. I'll try it out and adjust it if necessary.